pkg/performanceprofile/controller/performanceprofile_controller.go

```diff
@@ -328,7 +328,7 @@ func (r *PerformanceProfileReconciler) tunedProfileToPerformanceProfileForHypers
 		return nil
 	}
 	if len(cmList.Items) == 0 {
-		klog.Errorf("no performance profile ConfigMap found that matches label %s", hypershiftconsts.NodePoolNameLabel)
+		klog.V(4).InfoS("no performance profile ConfigMap found that matches label", "label", hypershiftconsts.NodePoolNameLabel)
```

Comment:
@Tal-or  is this just the string "label" ?

Reply:
That's the key: "label" the value is the nodePool name